skxray/core.py

```diff
@@ -729,6 +729,70 @@ def wedge_integration(src_data, center, theta_start,
     raise NotImplementedError()
 
 
+def radial_integration(pixel_size, calibrated_center, image_array,
+                       dist_sample, detector_size, wavelength,
+                       q_or_twotheta="Q"):
```

Comment:
I think it would be more sensible to rename this to something like `x_axis`, or `axis` or some such and then allow valid options of `q`, `two_theta`, `r`, and `d`. 

there should be an optional `bins` parameter that defaults to some sensible number

everything other than `calibrated_center` and `image_array` should be optional. That means that you'll have to ensure that you've got the correct input for each of the conversions

put image array first

remove `detector_size`